app/models/custom_value.rb

```diff
@@ -99,4 +110,10 @@ def validate_length_of_value
       errors.add(:value, :too_long, count: custom_field.max_length) if custom_field.max_length > 0 and value.length > custom_field.max_length
     end
   end
+
+  private
+
+  def strategy
+    @strategy = FORMAT_STRATEGIES[custom_field.field_format].new(self)
```

Comment:
Actually that is really what I meant to do... as in: I wanted to write `||=` and when reading your comment I could at first not see what you thought was wrong ^^